### PR TITLE
LPS-159104 Update permissions for account addresses

### PR DESCRIPTION
**Pull request by pei-jung**

### CODE REVIEW

**Comment by drewbrokke** [comment]
ci:reevaluate:1307401_3935

File: modules/apps/account/account-admin-web/src/main/java/com/liferay/account/admin/web/internal/frontend/taglib/servlet/taglib/AccountEntryAddressesScreenNavigationCategory.java [hunk 1]
```diff
@@ -67,7 +70,19 @@ public boolean isVisible(User user, AccountEntry accountEntry) {
 			return false;
 		}
 
-		return true;
+		if (AccountEntryPermission.contains(
+				PermissionCheckerFactoryUtil.create(user),
```
Comment (drewbrokke) [inline modules/apps/account/account-admin-web/src/main/java/com/liferay/account/admin/web/internal/frontend/taglib/servlet/taglib/AccountEntryAddressesScreenNavigationCategory.java:74]: Make this a variable? Not sure how expensive re-creating a permission checker is but I imagine it's enough to warrant not doing it more than necessary.

**Review by drewbrokke — CHANGES_REQUESTED** [review]
Just the one minor adjustment. Otherwise this looks good to go 👍

**Comment by epark18** [comment]
This failure was a result of a connection issue.

**Comment by drewbrokke** [comment]
See https://github.com/liferay-usm/liferay-portal/pull/679

**Pull request closed without merging**